packages/web/src/pages/App.js

```diff
@@ -474,7 +474,7 @@ class App extends Component {
           {this.props.showCookieBanner ? <CookieBanner /> : null}
           <Notice />
           <Navigator />
-          <div className={styles.draggableArea} />
+          {/* <div className={styles.draggableArea} /> */}
```

Comment:
should this be put back?

Reply:
Oh yes great call out. So weirdly this was blocking cypress from selecting the checkbox which I thought odd. Furthermore was wondering why this ui exists in the first place. Will follow up with you and maybe Julian on what the intention for this thing is